Client/Frontend/Fakespot/Components/CollapsibleCardContainer.swift

```diff
@@ -0,0 +1,169 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import UIKit
+
+class CollapsibleCardContainer: CardContainer, UIGestureRecognizerDelegate {
+    private struct UX {
+        static let verticalPadding: CGFloat = 8
+        static let horizontalPadding: CGFloat = 8
+        static let titleHorizontalPadding: CGFloat = 16
+        static let titleTopPadding: CGFloat = 16
+        static let chevronSize = CGSize(width: 20, height: 20)
+    }
+
+    enum ExpandButtonState {
+        case collapsed
+        case expanded
+
+        var image: UIImage? {
+            switch self {
+            case .expanded:
+                return UIImage(named: StandardImageIdentifiers.Large.chevronUp)?.withRenderingMode(.alwaysTemplate)
+            case .collapsed:
+                return UIImage(named: StandardImageIdentifiers.Large.chevronDown)?.withRenderingMode(.alwaysTemplate)
+            }
+        }
+
+        var toggle: ExpandButtonState {
+            switch self {
+            case .expanded:
+                return .collapsed
+            case .collapsed:
+                return .expanded
+            }
+        }
+    }
+
+    // MARK: - Properties
+    private var state: ExpandButtonState = .expanded
+
+    // UI
+    private lazy var rootView: UIView = .build { _ in }
+    private lazy var headerView: UIView = .build { _ in }
+    private lazy var containerView: UIView = .build { _ in }
+    private var containerHeightConstraint: NSLayoutConstraint?
+    private var tapRecognizer: UITapGestureRecognizer!
+
+    lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .headline, size: 17.0)
```

Comment:
Just realized, we'll need to move `DynamicFontHelper`, or some part of it we want to keep to `Common` 🤔 I'll create a task for this.